incubator/hnc/scripts/hier-set-parent

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 cat <<EOF | kubectl apply -f -
 apiVersion: hnc.x-k8s.io/v1alpha1
-kind: Hierarchy
+kind: HierarchyConfig
```

Comment:
Remember that this should be `HierarchyConfiguration`

Reply:
So noted, and changed.

Resolved.